mmv1/third_party/terraform/acctest/tgc_utils.go

```diff
@@ -0,0 +1,65 @@
+package acctest
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+)
+
+var CaiProductBackendNames = map[string]string{
+	"compute":         "compute",
+	"resourcemanager": "cloudresourcemanager",
+}
```

Comment:
I would recommend basing this on just the TF resource name instead of product -> service name. We can easily generate a comprehensive TF resource name -> API service name mapping, either generated from MMv1 or by reading the `resource` and `api_service_name` fields from metadata files like https://github.com/hashicorp/terraform-provider-google-beta/blob/main/google-beta/services/accessapproval/resource_organization_access_approval_settings_generated_meta.yaml